### PR TITLE
Submit data support

### PR DESCRIPTION
**Pull request by tstrass**
Full command to run new test:
```
brake crucible:execute[<server stu3 base url>,stu3,testscript-example-submit-data]
```

### CODE REVIEW

File: lib/tests/testscripts/scripts/reporting/submit-data.xml [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+	<teardown>
+		<action>
+			<operation>
+				<type>
+					<system value="http://hl7.org/fhir/testscript-operation-codes"/>
+					<code value="delete"/>
+				</type>
+				<resource value="Patient"/>
+				<description value="Delete the patient resource on the test server using the id of the Patient."/>
+				<targetId value="test-Patient-294"/>
```
Comment (mgramigna) [inline lib/tests/testscripts/scripts/reporting/submit-data.xml]: The `targetId` does not match the ID of the Patient that's created during `submit-data`. I assume we want these to match up so I'd replace this with the variable `${createPatientId}` that you defined.

**Review by mgramigna — CHANGES_REQUESTED** [review]

**Comment by mgramigna** [comment]
Other than my small comment, the rest of this looks good to me

**Review by mgramigna — CHANGES_REQUESTED** [review]
Missing a trailing `}`. 

Also, I'm not seeing anything actually get deleted from the server during teardown. I can easily see this being a problem with the engine and not the testscript itself. Perhaps the teardown portion is never even running in the first place? I doin't see any DELETE requests in the `cqf-ruler` logs. This is probably outside the scope of your PR and we can make a Jira (**EDIT:** Made a Jira for this)

**Comment by okeefm** [comment]
@tstrass needs a rebase. @mgramigna yeah let's make a Jira task to ensure that setup/teardown methods are run

**Comment by mgramigna** [comment]
@okeefm Sounds good. I made a Jira

**Review by okeefm — APPROVED** [review]
Looks good pending a rebase and fixing @mgramigna's change requests

**Review by mgramigna — APPROVED** [review]

**Pull request merged**